### ISSUE TITLE
qta-browser not starting with PyQt6==6.7.0 backend

### ISSUE THREAD

**Issue by o-murphy** [spyder-ide/qtawesome#261]
PySide6==6.7.0 still works

I think it's probleb with qtpy library, but can you check?

```bash
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "venv\Scripts\qta-browser.exe\__main__.py", line 7, in <module>
  File "venv\Lib\site-packages\qtawesome\icon_browser.py", line 334, in run
    browser = IconBrowser()
              ^^^^^^^^^^^^^
  File "venv\Lib\site-packages\qtawesome\icon_browser.py", line 108, in __init__
    toolbar.addWidget(self._copyButton)
TypeError: 'PySide6.QtWidgets.QBoxLayout.addWidget' called with wrong argument types:
  PySide6.QtWidgets.QBoxLayout.addWidget(QPushButton)
Supported signatures:
  PySide6.QtWidgets.QBoxLayout.addWidget(PySide6.QtWidgets.QWidget, int = 0, PySide6.QtCore.Qt.AlignmentFlag = Default(Qt.Alignment))

```

**Comment by o-murphy** [comment]
> Hey @o-murphy, thanks for reporting. This is a duplicate of issue [spyder-ide/qtpy#480](https://github.com/spyder-ide/qtpy/issues/480).
> 
> The problem is with PySide 6.7 because it broke star imports, which are used by Qtpy (as described on that issue).

But the issue title said i'm about QtPy6! i haven`t problem with PySide

**Comment by maflAT** [comment]
This issue seems to be fixed with PySide 6.7.1

**Comment by o-murphy** [comment]
> This issue seems to be fixed with PySide 6.7.1

Now my extension for your library works again
